web/themes/custom/cloudy/templates/content/events/node--event.html.twig

```diff
@@ -10,9 +10,9 @@
       <div class="row">
         <div class="col mb-4">
           <div class="event__date d-flex">
-            {% if content.field_end_date|render|striptags|trim is not empty %}
+            {% if content.field_start_date.0|render|striptags is not empty and content.field_end_date|render|striptags|trim is not empty %}
```

Comment:
Since Start Date is a required field you could refactor this to only check if Multiple days = true and End Date is not empty; else display start date.

Reply:
This works, so let's leave it for now and revisit as a part of Smart Date refactoring.